util/log/clog.go

```diff
@@ -513,8 +513,8 @@ func formatHeader(s Severity, now time.Time, threadID int32, file string, line i
 	n += buf.someDigits(n, int(threadID))
 	tmp[n] = ' '
 	n++
-	buf.Write(tmp[:n])
-	buf.WriteString(file)
+	_, _ = buf.Write(tmp[:n])
```

Comment:
I would keep `bytes:Write.*` in the whitelist and remove `_, _ =` wherever we can. The reason is that the whitelist will apply only when the receiver can be statically verified as a `bytes.Buffer`, while a habit of `_, _ =` could also discard errors from other implementations that ought to be checked.